.expeditor/run_windows_tests.ps1

```diff
@@ -0,0 +1,17 @@
+# Stop script execution when a non-terminating error occurs
+$ErrorActionPreference = "Stop"
```

Comment:
From my testing yesterday wehn this is set to 'Stop', we don't have to check `$lastexitcode` - the script will exit on non-zero return code.

Reply:
Great catch